src/python_pachyderm/client.py

```diff
@@ -62,24 +62,55 @@ def __init__(
         root_certs=None,
         transaction_id=None,
         tls=None,
+        use_default_host=True
     ):
         """
-        Creates a Pachyderm client.
+        Creates a Pachyderm client. If host and port are unset, checks the 
+        PACH_CONFIG env var for a path. If that's unset, it checks two 
+        file paths for a config file. If both files don't exist, a client
+        with default settings is created.
 
         Params:
 
         * `host`: The pachd host. Default is 'localhost', which is used with
         `pachctl port-forward`.
         * `port`: The port to connect to. Default is 30650.
-        * `auth_token`: The authentication token; used if authentication is
+        * `auth_token`: The authentication token. Used if authentication is
         enabled on the cluster. Defaults to `None`.
         * `root_certs`:  The PEM-encoded root certificates as byte string.
         * `transaction_id`: The ID of the transaction to run operations on.
         * `tls`: Specifies whether TLS should be used. If `root_certs` are
         specified, they are used; otherwise, we use the certs provided by
         certifi.
+        * `use_default_host`: Boolean for replicating `pachctl` behavior 
+        of searching for config. Default is True.
         """
 
+        # replicate pachctl behavior to searching for config if host and port are unset
```

Comment:
I think we should try to factor this out into a helper function and keep the __init__ clean.